controllers/schedule_controller.go

```diff
@@ -125,6 +125,24 @@ func (r *BackupScheduleReconciler) Reconcile(
 		return ctrl.Result{}, nil
 	}
 
+	// don't create schedule if an active restore exists
+	restoreName, err := r.isRestoreRunning(ctx, backupSchedule)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+	if restoreName != "" {
+		msg := "Restore resource " + restoreName + " is currently active, " +
+			"verify that any active restores are removed."
+		scheduleLogger.Info(msg)
+		backupSchedule.Status.Phase = v1beta1.SchedulePhaseFailedValidation
+		backupSchedule.Status.LastMessage = msg
+		// retry after failureInterval
```

Comment:
@birsanv backupSchedule status is set to SchedulePhaseFailedValidation and setting RequeueAfter: failureInterval